app/src/main/java/com/pileproject/drive/programming/visual/block/BlockFactory.java

```diff
@@ -33,20 +33,17 @@
 import static com.pileproject.drive.programming.visual.block.BlockCategory.SEQUENCE;
 
 /**
- * Factory that creates blocks
- *
- * @author <a href="mailto:tatsuyaw0c@gmail.com">Tatsuya Iwanari</a>
- * @version 1.0 18-June-2013
+ * Factory that creates blocks.
  */
 public class BlockFactory {
 
     private BlockFactory() {
         throw new AssertionError("This class cannot be instantiated");
     }
 
+    @SuppressWarnings("unchecked")
     private static <T extends BlockBase> Class<T> getClassForName(String className) throws RuntimeException {
         try {
-            // noinspection unchecked
```

Comment:
Thank you for your suggestion. I took the latter because the IDE-specific feature didn't suppress the warning like [this](https://travis-ci.org/PileProject/drive/builds/184892292#L2407) and it was annoying.